go.mod

```diff
@@ -20,53 +19,53 @@ require (
 	github.com/spf13/cobra v0.0.5
 	github.com/spf13/pflag v1.0.5
 	go.opencensus.io v0.21.0
-	golang.org/x/sys v0.0.0-20190826190057-c7b8b68b1456
+	golang.org/x/sys v0.0.0-20191022100944-742c48ecaeb7
 	gotest.tools v2.2.0+incompatible
-	k8s.io/api v0.0.0
-	k8s.io/apimachinery v0.0.0
-	k8s.io/client-go v0.0.0
+	k8s.io/api v0.18.4
+	k8s.io/apimachinery v0.18.4
+	k8s.io/client-go v0.18.4
 	k8s.io/klog v1.0.0
-	k8s.io/kubernetes v1.17.6
+	k8s.io/kubernetes v1.18.4
 )
 
-replace k8s.io/legacy-cloud-providers => github.com/kubernetes/kubernetes/staging/src/k8s.io/legacy-cloud-providers v0.0.0-20200520130745-d32e40e20d16
+replace k8s.io/legacy-cloud-providers => k8s.io/legacy-cloud-providers v0.18.4
 
-replace k8s.io/cloud-provider => github.com/kubernetes/kubernetes/staging/src/k8s.io/cloud-provider v0.0.0-20200520130745-d32e40e20d16
+replace k8s.io/cloud-provider => k8s.io/cloud-provider v0.18.4
 
-replace k8s.io/cli-runtime => github.com/kubernetes/kubernetes/staging/src/k8s.io/cli-runtime v0.0.0-20200520130745-d32e40e20d16
+replace k8s.io/cli-runtime => k8s.io/cli-runtime v0.18.4
 
-replace k8s.io/apiserver => github.com/kubernetes/kubernetes/staging/src/k8s.io/apiserver v0.0.0-20200520130745-d32e40e20d16
+replace k8s.io/apiserver => k8s.io/apiserver v0.18.4
 
-replace k8s.io/csi-translation-lib => github.com/kubernetes/kubernetes/staging/src/k8s.io/csi-translation-lib v0.0.0-20200520130745-d32e40e20d16
+replace k8s.io/csi-translation-lib => k8s.io/csi-translation-lib v0.18.4
 
-replace k8s.io/cri-api => github.com/kubernetes/kubernetes/staging/src/k8s.io/cri-api v0.0.0-20200520130745-d32e40e20d16
+replace k8s.io/cri-api => k8s.io/cri-api v0.18.4
```

Comment:
We shouldn't need these `replace` statements with these release tags.

Reply:
They're needed when k8s.io/kubernetes is required, because k/k's go.mod requires those libraries at v0.0.0 and replaces them with staging paths, so we still need to override that.

Projects that only require k8s.io libraries, not k8s.io/kubernetes itself, don't need replace statements.